docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #71.            for (start, stop) in cand_idx_chunks:
</pre><p>You might be able to omit the parentheses and do <code>for start, stop</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>